docs/Remote Control Guide/index.md

```diff
@@ -0,0 +1,185 @@
+
+### OnInteriorVehicleData
+
+OnInteriorVehicleData is a notification sent out by the HMI when an update is made to a remote control module. An app can subscribe to these notifications via GetInteriorVehicleData. This RPC will come with a `ModuleData` structure identifying the changed module and containing the control data object with the new state.
+
+View **OnInteriorVehicleData** in the [RPC Spec](https://github.com/smartdevicelink/rpc_spec#oninteriorvehicledata) or the [HMI Documentation](https://smartdevicelink.com/en/guides/hmi/rc/oninteriorvehicledata)
+
+### SetInteriorVehicleData
+
+SetInteriorVehicleData is used to set the values of a remote control module by passing in a `ModuleData` structure. The `moduleType` and `moduleId` fields are used to identify the targeted module, and the changes in the respective control data object are applied to that module.
+
+View **SetInteriorVehicleData** in the [RPC Spec](https://github.com/smartdevicelink/rpc_spec#setinteriorvehicledata) or the [HMI Documentation](https://smartdevicelink.com/en/guides/hmi/rc/setinteriorvehicledata)
+
+### OnRemoteControlSettings
+
+OnRemoteControlSettings is used to notify SDL when passengers of a vehicle change the remote control settings via the HMI. This includes allowing or disallowing remote control or changing the access mode that will be used for resource allocation.
+
+View **OnRemoteControlSettings** in the [HMI Documentation](https://smartdevicelink.com/en/guides/hmi/rc/onremotecontrolsettings)
+
+### OnRCStatus
+
+OnRCStatus is a notification sent out by SDL when an update is made to a remote control module's availability. When SDL either allocates a module to an app, or deallocates it from an app, SDL will send OnRCStatus to both the application and the HMI. This notification contains two lists, one describing the modules that are allocated to the application and the other describing the free modules that can be accessed by the application. This notification also contains an `allowed` parameter, which indicates to apps whether or not remote control is currently allowed. If `allowed` is false, both module lists will be empty.
+
+View **OnRCStatus** in the [RPC Spec](https://github.com/smartdevicelink/rpc_spec#onrcstatus) or the [HMI Documentation](https://smartdevicelink.com/en/guides/hmi/rc/onrcstatus)
+
+### GetInteriorVehicleDataConsent
+
+GetInteriorVehicleDataConsent is a request used to reserve remote control modules. If a module does not allow multiple access, only the application that requested consent first (excluding takeover situations described in the Consent section) will be able to interact with that module. This request requires a `moduleType` and an array of `moduleId`s to identify the target modules. Core will reply with an array of booleans indicating the consent for each requested `moduleId` where true signals allowed and vice versa.
```

Comment:
Could you add a link to the Consent section?

Reply:
added in https://github.com/smartdevicelink/sdl_core_guides/pull/61/commits/ccb44d882524cf70b78a5f47848832a09214ba5d